collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -247,9 +248,12 @@ def on_call(*args, **kwargs):
 ETC_PASSWD_PATH = '/etc/passwd'
 PROC_PATH = '/proc'
 
+IS_INSIDE_DOCKER = False
+
 if HOST_PREFIX:
```

Comment:
Is there ever a case where the HOST_PREFIX would be set but we not running in Docker, likewise not set but we actually are running in Docker? As much asking for my own curiosity.

Reply:
Yes, but i see no reason to set `NETDATA_HOST_PREFIX` var for any other reason. Checking well-known env vars is common practice, for example see https://github.com/kubernetes/client-go/blob/30548acd0a9e231d42d406433aef55b111b36240/rest/config.go#L483-L491

Reply:
> likewise not set but we actually are running in Docker

we mount host /proc, /sys, etc to /host/, see https://learn.netdata.cloud/docs/agent/packaging/docker#create-a-new-netdata-agent-container